modules/cnf-managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus.adoc

```diff
@@ -14,6 +14,28 @@ In the performance profile, `globallyDisableIrqLoadBalancing` is used to manage
 
 To achieve low latency for workloads, some (but not all) pods require the CPUs they are running on to not process device interrupts. A pod annotation, `irq-load-balancing.crio.io`, is used to define whether device interrupts are processed or not. When configured, CRI-O disables device interrupts only as long as the pod is running.
 
+[id="disabling-cpu-cfs-quota_{context}"]
+== Disabling CPU CFS quota
+
+To reduce CPU throttling for individual guaranteed pods create a performance profile with the annotation `cpu-quota.crio.io: "disable"`, this disables CPU completely fair scheduler (CFS) quota at the pod run time. An example pod specification snippet that illustrates this is below:
```

Comment:
```suggestion
To reduce CPU throttling for individual guaranteed pods, create a performance profile with the annotation `cpu-quota.crio.io: "disable"`. This annotation disables the CPU completely fair scheduler (CFS) quota at the pod run time. The following pod specification contains this annotation:
```

Reply:
> ! below adjective, adverb, preposition
Do not use to indicate a relative location in a document, as in "the information below".